### DISCUSSION TITLE
Is there a way to use the distributable with PyInstaller?

### DISCUSSION THREAD

**Discussion by JimiC** [indygreg/python-build-standalone#369] [Q&A]
I'm using `PDM` to install the `Python` version, which is using your distributable. When bundling my app with `PyInstaller`, running the executable fails, complaining of missing `Python` modules. If I use `PyEnv` for `Python` version management, the app works as expected.
Upon further inspection, I discovered that my cause of issues, are missing files (`*.so`) in `lib-dynload` of the installed `Python` version, when I use this distributable. Those files are present in the `PyEnv`'s one's and if I copy-paste them over, my bundle works just fine.

 Are those files missing intentionally?
 Is there a way to overcome the issue with `PyInstaller`?

**Comment by zanieb** [comment]
Hi! Which files are missing?

**Reply by JimiC** [reply]
PYENV:
![image](https://github.com/user-attachments/assets/b1202463-0844-433f-9d17-215871ff30a7)
PDM: 
![image](https://github.com/user-attachments/assets/01ab899e-22a5-460e-a33a-f027525979f5)

**Reply by JimiC** [reply]
> Hi! Which files are missing?

Any news on this?
